src/servers/Server_Zone/DebugCommand/DebugCommandHandler.cpp

```diff
@@ -519,5 +514,76 @@ void Core::DebugCommandHandler::serverInfo( char * data, Entity::Player& player,
 
 void Core::DebugCommandHandler::unlockCharacter( char* data, Entity::Player& player, boost::shared_ptr< DebugCommand > command )
 {
-   player.unlock( );
+   player.unlock();
+}
+
+void Core::DebugCommandHandler::script( char* data, Entity::Player &player, boost::shared_ptr< DebugCommand > command )
```

Comment:
this isn't really an issue with the PR per se as we're not doing this currently, but it'd be in my eyes a good code improvement:

in general for smart pointers we use an using directive to help with verbosity
try using DebugCommandPtr = boost::shared_ptr< DebugCommand >;
then you can use DebugCommandPtr in argument type

Reply:
check out Forwards.h